### ISSUE TITLE
Enhancement for handling of services - crashed services

### ISSUE THREAD

**Issue by apolkosnik** [crits/crits#844]
It seems that we're experiencing a common issue with services that never completed their run. The UI shows them as if they were still running.

In such a case, we can pre-populate the completion timestamp with a known value (possibly the epoch would do) before the service is kicked off, when the service, completes, it can update the timestamp, and it's business as usual. In case of the crashed services, we'll have an indicator that they have never completed, and we can act on it (number of retries, or some other action like adding it to a report or logging it).

**Comment by mgoffin** [comment]
I'd like to find another solution than to pre-populate the completion time with a fake value. That seems wrought with confusion for people who see stuff like Cuckoo as "in progress" but have a completion time.

**Comment by brlogan** [comment]
This might be related to the fix I submitted in PR #819.
If an exception occurs during execution of a service, and it isn't handled appropriately, the service never completes and you get the endless spinning circle in the Analysis tab of a TLO.
I added more robust exception handling for that one case, but I'm sure there are other places it could be improved.

**Comment by apolkosnik** [comment]
Yeah, the issue would be for the catastrophic failures which manage to escape the exception handler e.g. python/apache gets terminated. 
Another case is the failures from the past, that are left in this limbo state.

**Comment by brlogan** [comment]
Wouldn't looking for `status: started` tell you the same thing as a not updated `finish_date`?
I found a number of our stalled services using the `status` field and then re-ran them.
